src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -137,6 +241,51 @@ protected override void TriggerAnalysis(Request request, IIssueConsumer consumer
             {
                 TriggerAnalysisCallCount++;
             }
+
+            protected override void CallSubProcess(Action<Message> handleMessage, Request request, IAnalysisStatusNotifier analysisStatusNotifier, ISonarLintSettings settings, ILogger logger, CancellationToken cancellationToken)
+            {
+                if (onCallSubProcess == null)
+                {
+                    base.CallSubProcess(handleMessage, request, analysisStatusNotifier, settings, logger, cancellationToken);
+                }
+                else
+                {
+                    onCallSubProcess(handleMessage, request, analysisStatusNotifier, settings, logger, cancellationToken);
+                }
+            }
+        }
+
+        private class SubProcessSimulator
```

Comment:
This class is ugly but necessary of the way the CLangAnalyzer expects the stream handler to not return until all of the issues have been received.